scripts.js

```diff
@@ -15,4 +15,251 @@ const navLinks = document.querySelectorAll('.nav-item');
+  mobileImage.src = card.deskImage;
+  content.appendChild(mobileImage);
+  const DeskImage = document.createElement('img');
+  DeskImage.className = 'desktop_image  enl';
+  DeskImage.src = card.deskImage;
+  content.appendChild(DeskImage);
+  const ContentDiv = document.createElement('div');
+  ContentDiv.className = 'flex_content pop';
+  content.appendChild(ContentDiv);
+  const heading3 = document.createElement('h3');
+  canopyDiv.appendChild(heading3);
+  heading3.className = 'modalHeader';
+  heading3.textContent = card.heading3;
+  const paragraph = document.createElement('p');
+  paragraph.className = 'card-content pop_paragh';
+  ContentDiv.appendChild(paragraph);
+  paragraph.textContent = card.paragraph;
+  const list1 = document.createElement('ul');
+  list1.className = 'class-content ul1_modal ';
+  canopyDiv.appendChild(list1);
+  const classes = ['canopy', 'back-end', 'year'];
+  for (let j = 0, i = 0; j < card.list1.length - 1; j += 1, i += 1) {
+    const list1Elements = document.createElement('li');
+    list1Elements.className = classes[i];
+    list1Elements.textContent = card.list1[j];
+    list1.appendChild(list1Elements);
+
+    if (j < card.list1.length - 2) {
+      const list1items = document.createElement('l1');
+      list1items.className = 'dot';
+      const dotz = document.createElement('img');
+      dotz.src = li4;
+      list1items.appendChild(dotz);
+      list1.appendChild(list1items);
+    }
+  }
+  const languages = document.createElement('div');
+  languages.className = 'languages poplang';
+  ContentDiv.appendChild(languages);
+  const langList = document.createElement('ul');
+
+  languages.appendChild(langList);
+  for (let i = 0; i < card.list2.length; i += 1) {
+    const list2Items = document.createElement('li');
+    list2Items.className = 'lang--names';
+    list2Items.textContent = card.list2[i];
+    langList.appendChild(list2Items);
+  }
+  const list2Btndiv = document.createElement('div');
+  list2Btndiv.className = 'list2_btn';
+  languages.appendChild(list2Btndiv);
+  const btnDiv = document.createElement('div');
+  btnDiv.className = 'btnDiv';
+  list2Btndiv.appendChild(btnDiv);
+  const liveLink = document.createElement('a');
+  liveLink.href = card.cardButton1;
+  const cardButton1 = document.createElement('div');
+  cardButton1.className = 'see-more';
+  liveLink.appendChild(cardButton1);
+  const btnimg = document.createElement('img');
+  btnimg.src = './images/btnIcon.svg';
+  cardButton1.appendChild(btnimg);
+  const btntext = document.createElement('p');
+  btntext.textContent = 'See Live';
+  cardButton1.appendChild(btntext);
+  btnDiv.appendChild(liveLink);
+
+  const source = document.createElement('a');
+  source.href = card.cardButton2;
+  const cardButton2 = document.createElement('div');
+  cardButton2.className = 'see-more';
+  source.appendChild(cardButton2);
+  const btnimg2 = document.createElement('img');
+  btnimg2.src = './images/btn.svg';
```

Comment:
:red_circle: [Requested Change]  Kindly make sure your buttons' images are displaying properly. I am unable to view them on the popup
This is what I mean :arrow_down: 

Your Design :paintbrush: 
![popuimage](https://user-images.githubusercontent.com/26736582/204087332-c29698ba-991b-4380-82ef-fa11208a362f.png)

Figma Design :paintbrush: 
![btnfigma](https://user-images.githubusercontent.com/26736582/204087356-9050b373-cf4a-4447-b97e-32346b6177c0.png)